forge-game/src/main/java/forge/game/zone/Zone.java

```diff
@@ -120,12 +120,11 @@ public void add(final Card c, Integer index, final Card latestState) {
         if (zoneType != ZoneType.Battlefield) {
             c.setTapped(false);
         }
-
-        // Do not add Tokens to other zones than the battlefield. (unless it's a copy of a card 707.12)
-        // But Effects/Emblems count as Tokens too, so allow Command too.
-        if ((zoneType == ZoneType.Battlefield || !c.isToken()) || (zoneType == ZoneType.Stack && c.getCopiedPermanent() != null)) {
+        //Emblem/Effects are Immutable..
+        if (!c.isToken() || !c.isImmutable())
```

Comment:
the condition seems wrong? Aren't each card either not a token or not a Immutable?
each token is not immutable
and each immutable is not a token?

Reply:
Hmm I thought effect are immutable and treated as token?

Reply:
i think it was once, but i changed it that Effects and Emblems aren't treated as Token anymore

Reply:
Imo we can just always setZone without restrictions

that should make that part equivalent to the old behaviour before #93

(previously c=copied in GameAction so tokens Zone was also changed)

Reply:
yeah i think that might be okay